betfairlightweight/enums.py

```diff
@@ -236,3 +236,266 @@ class StreamingOrderType(Enum):
+                       Filters out orders which have not been voided. The dateRange filter (if specified) is applied to the voided date.
+    """
+    BetId = 'BY_BET'
+    MarketId = 'BY_MARKET'
+    MatchTime = 'BY_MATCH_TIME'
+    PlaceTime = 'BY_PLACE_TIME'
+    SettleTime = 'BY_SETTLED_TIME'
+    VoidTime = 'BY_VOID_TIME'
+
+
+class SortDir(Enum):
+    """
+    Sorting by times enum.
+    :var EARLIEST_TO_LATEST: Order from earliest value to latest e.g. lowest betId is first in the results.
+    :var LATEST_TO_EARLIEST: Order from the latest value to the earliest e.g. highest betId is first in the results.
+    """
+    TimeAscending = 'EARLIEST_TO_LATEST'
+    TimeDescending = 'LATEST_TO_EARLIEST'
+
+
+class BetStatus(Enum):
+    """
+    Set bet status to filter for.
+    :var SETTLED: A matched bet that was settled normally
+    :var VOIDED: A matched bet that was subsequently voided by Betfair, before, during or after settlement
+    :var LAPSED: Unmatched bet that was cancelled by Betfair (for example at turn in play).
+    :var CANCELLED: Unmatched bet that was cancelled by an explicit customer action.
+    """
+    Settled = 'SETTLED'
+    Voided = 'VOIDED'
+    Lapsed = 'LAPSED'
+    Cancelled = 'CANCELLED'
+
+
+class GroupBy(Enum):
+    """
+    Grouping of P&L reporting
+    :var EVENT_TYPE: A roll up of settled P&L, commission paid and number of bet orders, on a specified event type
+    :var EVENT: A roll up of settled P&L, commission paid and number of bet orders, on a specified event
+    :var MARKET: A roll up of settled P&L, commission paid and number of bet orders, on a specified market
+    :var SIDE: An averaged roll up of settled P&L, and number of bets, on the specified side of a specified selection within a specified market, that are either settled or voided
+    :var BET: The P&L, commission paid, side and regulatory information etc, about each individual bet order
+    """
+    EventType = 'EVENT_TYPE'
+    Event = 'EVENT'
+    Market = 'MARKET'
+    Side = 'SIDE'
+    Bet = 'BET'
+
+
+class OrderType(Enum):
+    """
+    Define order type in sending orders to exchange.
+    :var LIMIT: A normal exchange limit order for immediate execution
+    :var LIMIT_ON_CLOSE: Limit order for the auction (SP)
+    :var MARKET_ON_CLOSE: Market order for the auction (SP)
+    """
+    LimitOrder = 'LIMIT'
+    LimitOnClose = 'LIMIT_ON_CLOSE'
+    MarketOnClose = 'MARKET_ON_CLOSE'
+
+
+class PersistenceType(Enum):
+    """
+    Description supplied to tell exchange how to handle bet.
+    :var LAPSE: Lapse the order when the market is turned in-play
+    :var PERSIST: Persist the order to in-play. The bet will be place automatically into the in-play market at the start of the event.
+    :var MARKET_ON_CLOSE: Put the order into the auction (SP) at turn-in-play
+    """
+    Kill = 'LAPSE'
+    Keep = 'PERSIST'
+    ClosePrice = 'MARKET_ON_CLOSE'
+
```

Comment:
Agreed kill is confusing, maybe not match though as it is also misleading lapse or cancel probably most logical.